src/fiskaltrust.ifPOS/v1/de/IDESSCD.cs

```diff
@@ -92,5 +92,11 @@ public interface IDESSCD
         [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/endexportsession", Method = "POST")]
 #endif
         Task<EndExportSessionResponse> EndExportSessionAsync(EndExportSessionRequest request);
+
+        [OperationContract]
+#if WCF
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/echo", Method = "POST")]
+#endif
+        Task<TseEchoResponse> EchoAsync(TseEchoRequest request);
```

Comment:
I think we should either call this `EchoRequest` or `ScuEchoRequest` (same for response) since we are not really pinging the TSE

Reply:
Good point, i renamed it to `ScuEcho...` :)